packages/client/src/byline.ts

```diff
@@ -26,11 +26,11 @@ import util from 'util'
 
 // convenience API
 export default function byline(readStream, options?: any) {
-  return module.exports.createStream(readStream, options)
+  return createStream(readStream, options)
 }
 
 // basic API
-module.exports.createStream = function (readStream, options) {
+export function createStream(readStream, options) {
```

Comment:
Fun fact: since this is treated as an ES module by esbuild, it just ignored `module.exports` and didn't transform it in any way, so previously `createStream` and `LineStream` were defined globally on `module.exports` of the whole bundle:

```
Welcome to Node.js v18.1.0.
Type ".help" for more information.
> require('./packages/cli/build/index.js')
{
  createStream: [Function (anonymous)],
  LineStream: [Function: LineStream]
}
```

Reply:
🤣🙈